service/healthz/k8s/error.go

```diff
@@ -0,0 +1,12 @@
+package k8s
+
+import (
+	"github.com/juju/errgo"
+)
+
+var invalidConfigError = errgo.New("invalid config")
```

Comment:
I noticed I forgot to fix this before merge. Done in https://github.com/giantswarm/microendpoint/commit/13910f52552f2290e2b6c527e0e51d248d09c8fc.